pkg/models/system_intake.go

```diff
@@ -239,10 +239,10 @@ type RelatedSystemIntake struct {
 	RelatedRequestID uuid.UUID `db:"related_request_id"`
 }
 
-func (s *RelatedSystemIntake) GetMappingID() uuid.UUID {
+func (s RelatedSystemIntake) GetMappingKey() uuid.UUID {
```

Comment:
Would you be able to just chuck a comment explaining what we use `RelatedSystemIntake` for? I think it would help in understandiong these mapping functions if we have comments on the structs/types that they're for!

Reply:
(Couldn't leave the review comment on the actual struct type in this PR)